distributions/README.md

```diff
@@ -0,0 +1,20 @@
+# Criteria for Supported Distributions
+
+Support for a distribution implies all the duties covered by the Approver and Maintainer role requirements. In addition, support means that the Collector SIG is the owner and maintainer of the binaries/images of the different Collector distributions and is responsible for the pipeline that produces those artifacts.
+
+The Collector SIG must support at least one distribution that is recommended for production which includes support for Prometheus, Jaeger, Zipkin, and OpenCensus.
+
+Distributions supported by the Collector SIG should fulfill the following criteria:
+
+1. Serve a specific purpose and those purposes should have minimal overlap.
+2. Meet general needs and not be too niche.
+3. Only target the needs of the OpenTelemetry project.
+4. May be focused on development and proof of concept use cases.  The distribution should clearly indicate whether the Collector SIG recommends the distribution be used in Production environments.
+5. Must only include components from the `opentelemetry-collector` and `opentelemetry-collector-contrib` repositories.
+6. Have a clearly defined list of criteria for which components are included.
```

Comment:
Before we merge, can we have tracking issues to ensure we define these lists for our existing distributions?

Reply:
@djaglowski @evan-bradley we're currently using https://github.com/open-telemetry/opentelemetry-collector-releases/issues/360 to track the criteria building.  The discussion for Core continues but contrib has stalled, so once this is merged I plan to do a PR proposing Contrib's criteria.